docs/testing/testplan.md

```diff
@@ -0,0 +1,355 @@
+**Expected results**:
+
+1. The wallet owner receives an interaction prompt from their wallet to authorize the connection and access to the amount.
+2. On accepting, you get shown the message "Your wallet is now successfully connected to the extension."
+
+##### Make continuous payments
+
+**Test ID**: 2  
+**Description**: Send continuous WM payments to a monetized website  
+**Risk**: R3 (security of funds)  
+**Preconditions**:
+
+1. The extension is connected to your wallet.
+2. The extension has a positive remaining balance.
+3. Your wallet balance is equal to or greater than the extension’s remaining balance.
+
+**Steps**:
+
+1. Visit a monetized website. Refer to the [Test Data](#Test-Data) section to explore different WM conditions for websites.
+2. Visit a non-monetized website. Open the extension to observe its available options.
+
+**Expected results**:
+
+| Web monetized websites                                                                                                                                                                                                                                                          | Non monetized websites                                                          |
+| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :------------------------------------------------------------------------------ |
+| Extension icon: active (full color), with a green tick                                                                                                                                                                                                                          | Extension icon: active (full color), but with a red X                           |
+| Opening the extension displays a **rate of pay** slider:<ul><li>On the left, the current hourly rate in the currency of the wallet.</li><li>The default is equivalent to 0.60 USD.</li><li>The remaining balance, updated in near real-time (i.e. every few seconds).</li></ul> | Opening the extension displays <ul><li>This website is not monetized.</li></ul> |
+
+##### Pay one-time when extension and wallet have enough funds
+
+**Test ID**: 3  
+ **Description**: Send a one-time payment to a monetized website when the remaining balance for the extension is sufficient for the payment, and the wallet has sufficient funds  
+ **Risk**: R3 (security of funds) and R6 (control my payments)  
+ **Preconditions**:
+
+1.  The extension is connected to your wallet.
+2.  The extension has a positive remaining balance.
+3.  Your wallet balance is equal to or greater than the extension’s remaining balance.
+
+**Steps**:
+
+1. Visit a monetized website. Refer to the [Test Data](#Test-Data) section to explore different WM conditions for websites.
```

Comment:
Reminder that there's mention of Test Data here. Testers probably don't care about the different conditions if we're asking them to specifically visit a web monetized site.

Reply:
Yep, that's a fair point. I've updated the wording like this: 

> Visit a monetized website. The Test Data section lists monetized websites that you can visit.